results-processor/wptreport.py

```diff
@@ -505,6 +498,33 @@ def serialize_gzip(self, filepath):
         self.write_gzip_json(filepath, self._report)
 
 
+def _channel_to_labels(browser: str, channel: str) -> Set[str]:
+    """Maps browser-specific channels to well-known labels.
```

Comment:
Bit of a nit, but given that we put `channel` into `labels` to begin with, and could end up just exiting without doing anything more (say for `channel=foo`) it's not really 'to well-known labels'. It more attempts to extend the set of labels to include well-known ones.

It also doesn't align them to RELEASE_CHANNEL_LABELS - canary maps to nightly, and neither of those are in RELEASE_CHANNEL_LABELS ? (To be fair, there's an issue for those so maybe thats ok).

Reply:
> Bit of a nit, but given that we put channel into labels to begin with, and could end up just exiting without doing anything more (say for channel=foo) it's not really 'to well-known labels'. It more attempts to extend the set of labels to include well-known ones.

Maybe

```suggestion
    """Maps a browser-specific channel to labels.

    The original channel is always preserved as a label. In addition,
    well-known aliases of browser-specific channels are added.
```

> It also doesn't align them to RELEASE_CHANNEL_LABELS - canary maps to nightly, and neither of those are in RELEASE_CHANNEL_LABELS ? (To be fair, there's an issue for those so maybe thats ok).

No, it doesn't, and that's why I didn't use the same term (but "well-known" instead). This is point 2 in the PR description, which can be split into a separate change.

Reply:
SGTM